Notifications/readme.md

```diff
@@ -1,55 +0,0 @@
-Welcome to the Notifications section of the toolkit! This contains the Notifications library, including the object model for tile, toast, and badge XML (previously called NotificationsExtensions).
```

Comment:
Why is this deleted?

Reply:
The whole folder was deleted, not sure where that should go?